### PR TITLE
multiple doc changes + bugfixes

### PR DESCRIPTION
**Pull request by vivekbhr**
Mostly doc changes in XML and .rst files..
Bugfixes for options not working in XMLs.
Minor bugfixes in  deeptools.

### CODE REVIEW

File: galaxy/wrapper/bamCorrelate.xml [hunk 1]
```diff
@@ -102,7 +102,15 @@
     <help>
 <![CDATA[
 
+<<<<<<< HEAD
+.. image:: $PATH_TO_IMAGES/QC_bamCorrelate_humanSamples.png
```
Comment (bgruening) [inline galaxy/wrapper/bamCorrelate.xml]: @vivekbhr here you have a merge conflict.

**Comment by bgruening** [comment]
LGTM! Let me know when this is out of WIP.

**Comment by vivekbhr** [comment]
@bgruening :+1:

**Comment by dpryan79** [comment]
@vivekbhr Go ahead and merge once the conflicts with Thomas' changes are resolved.

**Comment by dpryan79** [comment]
Sorry, the last merge conflict was my fault :P

**Comment by bgruening** [comment]
Green!

**Comment by bgruening** [comment]
Thanks a lot @vivekbhr! Great PR!

**Pull request merged**